course/steps/03-clients-analytics/en.md

```diff
@@ -60,7 +61,14 @@ In this step, we will implement the Anaylitcs client. So,
 
    > The method that you've just created will get the necessary data for this application: an array of objects that have two fields: `slug`, a string that represents the product ID and `liveUsers`, a number that is the quantity of users visualizing this product - which are the fields in the interface.
 
-4. With your analytics client already implemented, it's necessary to declare it as one of the clients in the `Clients` class, so it will be accessible using the `Context` that we've talked about at the beginning of this step.
+4. The `_v/live-products` API endpoint we call above needs the app `mocked-analytics` to run, or your `getLiveUsers` method will not see anything there. Clone a copy of the [`mocked-analytics` repository](https://github.com/vtex-apps/mocked-analytics/) locally. 
```

Comment:
This dependency is not a problem when using the `appliancetheme` account, because we've already installed it there. I'm guessing that, in your case, you're using your own account and this app is not installed. 
I believe we need to change this step to emphasize that those who are running on the `appliancetheme` account can skip this step. cc @gris @juliarocha

Reply:
What do you think about changing the text a bit? Instead of linking another app and having three apps running at the same time, you can install this app on your workspace.

```suggestion
4. The `_v/live-products` API endpoint we call above needs the app `mocked-analytics` to run, or your `getLiveUsers` method will not see anything there. **In case of not running this application on `appliancetheme` account**, you will have to install the `mocked-analytics` app on your workspace. To do that, you can run `vtex install vtex.mocked-analytics`.
```